connectors/access_control.py

```diff
@@ -0,0 +1,52 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License 2.0;
+# you may not use this file except in compliance with the Elastic License 2.0.
+#
+
+ACCESS_CONTROL = "_allow_access_control"
+
+
+def prefix_identity(prefix, identity):
+    if prefix is None or identity is None:
+        return None
```

Comment:
What if one is passed and the other is none?

Would it be nice to warn in this case?